vnext/ReactWindowsCore/WebSocketModule.h

```diff
@@ -23,23 +26,46 @@ class WebSocketModule : public facebook::xplat::module::CxxModule
   };
 
   WebSocketModule();
+
+  /// <summary>
+  /// <see cref="facebook::xplat::module::CxxModule::getName" />
+  /// </summary>
   std::string getName();
+
+  /// <summary>
+  /// <see cref="facebook::xplat::module::CxxModule::getConstants" />
+  /// </summary>
   virtual std::map<std::string, folly::dynamic> getConstants();
+
+  /// <summary>
+  /// <see cref="facebook::xplat::module::CxxModule::getMethods" />
+  /// </summary>
+  /// <remarks>See See react-native/Libraries/WebSocket/WebSocket.js</remarks>
   virtual std::vector<Method> getMethods();
 
 private:
+  /// <summary>
+  /// Notifies an event to the current React Instance.
+  /// </summary>
```

Comment:
>/// </summary> [](start = 2, length = 14)

[nit] Doc comments for parameters (here and below)?

Reply:
Remaining parameters will be further documented in a subsequent change.